CMakeLists.txt

```diff
@@ -95,6 +106,32 @@ qsk_setup_install()
 add_subdirectory(src)
 add_subdirectory(skins)
 
+include(CMakePackageConfigHelpers)
+
+# write QSkinnyConfigVersion.cmake file for cmake import
+write_basic_package_version_file(
+    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}ConfigVersion.cmake
+    VERSION ${PACKAGE_VERSION}
+    COMPATIBILITY AnyNewerVersion)
+
+# Copy QSkinnyConfig.cmake to build dir
+configure_file(${QSK_CMAKE_DIR}/${PACKAGE_NAME}Config.cmake
+    ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}Config.cmake
+    COPYONLY)
+
+# install QSkinnyConfig.cmake and QSkinnyConfigVersion.cmake
+# and QskTools.cmake file to lib/cmake/QSkinny directory
+install(
+    FILES
+        ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}Config.cmake
+        ${CMAKE_BINARY_DIR}/_QSkinny/${PACKAGE_NAME}ConfigVersion.cmake
+        ${QSK_CMAKE_DIR}/QskTools.cmake
```

Comment:
@lgtmak we could move this `${QSK_CMAKE_DIR}/QskTools.cmake` install into `tools/CMakeLists.txt` 

```cmake
install(FILES ${QSK_CMAKE_DIR}/QskTools.cmake DESTINATION ${PACKAGE_LOCATION})
```

and include it optionally in `QSkinnyConfig.cmake`

Reply:
Addressed in #359